planning/work-items/other-documents-sync/arch.md

```diff
@@ -112,33 +112,47 @@ Continuation of https://github.com/w3c/wot-thing-description/issues/2120
+
 - [ ] 29. Producers of TDs MUST ensure that no Private Security Data is included in TDs. ([arch-security-consideration-no-private-security-data](https://www.w3.org/TR/wot-architecture11/#arch-security-consideration-no-private-security-data))
+  - A3: Fits to https://w3c.github.io/wot-thing-description/#behavior-security
+  - Should be merged with 27 and 28. 29 covers the other two. There is no way to violate one but fullfil the other two. Then 27 and 28 will be A2.
 
 #### 10.1.2 Thing Description Communication Metadata Risk
 
 - [ ] 30. Whenever possible, TD creators SHOULD use the vetted communication metadata provided in the WoT Binding Templates. ([arch-security-consideration-communication-binding](https://www.w3.org/TR/wot-architecture11/#arch-security-consideration-communication-binding))
 
+  - D (also the meaning is not clear, which is a sign for not making it normative)
+
 - [ ] 31. When generating TDs for an IoT Platform not covered by the WoT Binding Templates, TD creators SHOULD ensure that all the security requirements of the IoT Platform are satisfied. ([arch-security-consideration-communication-platform](https://www.w3.org/TR/wot-architecture11/#arch-security-consideration-communication-platform))
+  - D (also the meaning is not clear, which is a sign for not making it normative)
 
 ### 10.2 WoT Scripting API Risks
 
 #### 10.2.1 Cross-Script Security Risk
 
 - [ ] 32. The WoT Runtime SHOULD perform isolation of script instances and their data from each other in cases when scripts handle sensitive data. ([arch-security-consideration-isolation-sensitive](https://www.w3.org/TR/wot-architecture11/#arch-security-consideration-isolation-sensitive))
 
+  - D (can also be B and in Scripting API but this is not limited to scripting api implementations)
+
 - [ ] 33. Similarly, the WoT Runtime implementation SHOULD perform isolation of WoT Runtime instances and their data from each other if a WoT device has more than one tenant. ([arch-security-consideration-isolation-tenants](https://www.w3.org/TR/wot-architecture11/#arch-security-consideration-isolation-tenants))
+  - D (can also be B and in Scripting API but this is not limited to scripting api implementations)
 
 #### 10.2.2 Physical Device Direct Access Risk
 
 - [ ] 34. The WoT Runtime SHOULD NOT directly expose low-level device hardware interfaces to the script developers. ([arch-security-consideration-avoid-direct](https://www.w3.org/TR/wot-architecture11/#arch-security-consideration-avoid-direct))
 
+  - D (can also be B and in Scripting API but this is not limited to scripting api implementations)
+
 - [ ] 35. A WoT Runtime implementation SHOULD provide a hardware abstraction layer for accessing the low-level device hardware interfaces. ([arch-security-consideration-use-hal](https://www.w3.org/TR/wot-architecture11/#arch-security-consideration-use-hal))
+  - D (can also be B and in Scripting API but this is not limited to scripting api implementations)
```

Comment:
Categorization is fine.

BTW, I don't really see any difference to 34 🤷‍♂️

Reply:
I would say that 35 is what you should do to avoid 34. 35 is more of a guideline so we should keep that.

Note: Technically by not exposing anything device-related, one would achieve 34. Like adding two numbers that happen purely in software.